pom.xml

```diff
@@ -398,57 +371,6 @@
   </build>
 
   <profiles>
-    <profile>
-      <id>linux32</id>
-      <activation>
-        <os>
-          <family>Unix</family>
-          <arch>x86</arch>
-        </os>
-      </activation>
-      <dependencies>
-        <dependency>
-          <groupId>org.eclipse.swt</groupId>
-          <artifactId>org.eclipse.swt.gtk.linux.x86</artifactId>
-          <version>4.5.1</version>
-        </dependency>
-      </dependencies>
-    </profile>
-    <profile>
```

Comment:
Removing 32 bit OS profiles - yes. I know we don't officially support Linux but there might be users who are dependent on `linux64`, maybe not. We can remove it or not, your call.